main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: baby
+//  action 1: the baby cries
+baby.cry();
+//  action 2: the baby eats
+baby.eat();
+//  action 3: the baby sleeps
+baby.sleep();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: cat
+//  action 1: the cat meows
+cat.meow();
+//  action 2: the cat purrs
+cat.purr();
+//  action 3: the cat sleeps
+cat.sleep();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bird
+//  action 1: the bird sings
+bird.sing();
+//  action 2: the bird flies
+bird.fly();
+//  action 3: the bird eats worms
+bird.eatWorm();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: software
+//  action 1: the software crashes
+software.crash();
+//  action 2: the software runs
+software.run();
+//  action 3: the software initiates an update
+software.initiateUpdate();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: robot
+//  action 1: the robot walks;
+robot.walk();
+//  action 2: the robot hugs;
+robot.hug();
+//  action 3: the robot reads a book;
+robot.readBook();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: fingers
+//  action 1: the fingers bend
+fingers.bend();
+//  action 2: the fingers play the keys
+fingers.playKeys();
+//  action 3: the fingers turn on the light switch
+fingers.turnOnLightSwitch();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: person
+//  action 1: the person sits
+person.sit();
+//  action 2: the person watches tv
+person.watchTv();
+//  action 3: the person falls asleep
+person.fallAsleep();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bufferfly
+//  action 1: the butterfly flies
+butterfly.fly();
+//  action 2: the butterfly polinates
+butterfly.polinate();
+//  action 3: the butterfly hides
+butterfly.hide();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: car
+//  action 1: the car moves
+car.move();
+//  action 2: the car consumes fuel
+car.consumeFuel();
+//  action 3: the car stops
+car.stop();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.